mlir/utils/jenkins/miopen-tests/miopen_validate.sh

```diff
@@ -12,7 +13,7 @@ function usage() {
     cat <<END
 $0: [-d | --direction] DIR [-t | --dtype] [fp16 | fp32]
 [-l | --layout] LAYOUT [-x | --xdlops] [-X | --no-xdlops (default)]
-[--driver DRIVER (default bin/MIOpenDriver)]
+[--no-tuning (default)] [--driver DRIVER (default bin/MIOpenDriver)]
```

Comment:
Could we get this into a `[ --tuning  | --no-tuning (default) ]` for the docstring?

Reply:
Sure, done in 4bb1356